pkg/k8s/apps/interface.go

```diff
@@ -46,3 +46,20 @@ type App interface {
 
 	Divert(username string) App
 }
+
+type StaticPod interface {
```

Comment:
@Charlie17Li you also need to add the code to implement the interface.
For example, this is the implementation for deployments:
https://github.com/okteto/okteto/blob/master/pkg/k8s/apps/deployments.go

Reply:
OK👌🏻